index.js

```diff
@@ -21,15 +22,128 @@ const IGNORED_TASK_DEFINITION_ATTRIBUTES = [
   'registeredBy'
 ];
 
+//Code to run task outside of a service aka (also known as) a one-off task
+async function runTask(ecs, capacityProviderStrategy, clusterName, taskDefArn, waitForMinutes) {
+  core.info('Running task')
+
+  const waitForTask = core.getInput('wait-for-task-stopped', { required: false }) || 'false';
+  const startedBy = core.getInput('run-task-started-by', { required: false }) || 'GitHub-Actions';
+  const launchType = core.getInput('run-task-launch-type', { required: false }) || 'FARGATE';
```

Comment:
Do we see any merit in validating that the `launchType` is either of the required options, or would that be enforced elsewhere(non server-side)?

Reply:
If im understanding your questions correctly, yes we do need to keep it as a required option since the other features that was added was capacity provider. Depending on the users goal, they can pick either the launch type option or capacity provider (using both is not allowed).